node-notes-cli/app.js

```diff
@@ -0,0 +1,42 @@
+const data = require('./data.json');
+const fs = require('fs');
+
+const notesStringify = JSON.stringify(data.notes, null, 2);
+const objectNotes = JSON.parse(notesStringify);
+
+const dataStringify = JSON.stringify(data, null, 2);
+const dataObject = JSON.parse(dataStringify);
```

Comment:
You can skip most of this. `data` is the whole `dataObject` already and `data.notes` is `objectNotes`.